templates/main.Dockerfile

```diff
@@ -20,4 +20,4 @@ LABEL \
       io.k8s.display-name="Red Hat OpenShift Serverless Bundle" \
       com.redhat.openshift.versions="__OCP_TARGET_VLIST__" \
       com.redhat.delivery.operator.bundle=true \
-      com.redhat.delivery.backport=true
+      com.redhat.delivery.backport=false
```

Comment:
Just curious. What does this mean?

Reply:
We do not need to backport the bundle to the appregistry format since 1.11 is targeting 4.6.

Reply:
I thought we will try to use separate bundles to our advantage. What I mean is to deploy specific versions of Serverless to specific OCP 4.6+ versions, and having channel called `stable` in all of them. That should be possible with the new bundle format, right?

https://issues.redhat.com/browse/SRVCOM-886

Reply:
@cardil this is not the right place to resolve this. I don't remember discussions about changing the channel strategy in this way from our program calls. Maybe you can bring it up there.

Reply:
Sure. I think the same.

Reply:
Per https://issues.redhat.com/browse/RFE-723 and discussions I've had with Shawn, we'll be able to use a `stable` channel beginning with OCP 4.7.